jerry-core/parser/js/js-parser-statm.c

```diff
@@ -442,6 +442,18 @@ parser_parse_var_statement (parser_context_t *context_p) /**< context */
 
 #if ENABLED (JERRY_ES2015)
   uint8_t declaration_type = context_p->token.type;
+
+  scanner_location_t start_location;
+  scanner_get_location (&start_location, context_p);
+
+  lexer_next_token (context_p);
+
+  if (parser_parse_initializer (context_p, PARSER_PATTERN_BINDING))
+  {
+    return;
+  }
+
+  scanner_set_location (context_p, &start_location);
```

Comment:
While this is not part of the `while (true)` loop below? Also, I would use check next character instead of the heavyweight next_token + rollback.